prometheus-exporters/vrops-exporter/alerts/host.alerts

```diff
@@ -66,3 +66,27 @@ groups:
     annotations:
       description: "Failover Host {{ $labels.hostsystem }} has Virtual Machines on it. Free up the host. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
       summary: "Failover Host {{ $labels.hostsystem }} has Virtual Machines on it. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+  - alert: HostDiskspaceUsageWarning
```

Comment:
I would prefer `DiskSpace` with an uppercase S.

Reply:
Also for the other alerts.

Reply:
yeah, you are right. It is disk space, not diskspace